### DISCUSSION TITLE
MeshCentral server on openwrt 23 works fine

### DISCUSSION THREAD

**Discussion by leleb** [Ylianst/MeshCentral#5889] [Show and tell]
Just let you know that openwrt 23 has node and node-npm packages available for x86 and aarch64 devices with enough resources (no mips or mipsel) thus I successfully installed MeshCentral with Ne DB in an openwrt router. 
Only issues is that MeshCentral -install
 fails to set the service at startup. It could be fixed in future, however it is not difficult to do it manually (like the agent do)

**Comment by Ylianst** [comment]
Nice that you mention this. I am a big fan of OpenWRT, but run it on a Linksys WRT1900ACS. I should give it a try on a x86.

**Comment by leleb** [comment]
It should also run in your wrt1900acs with extroot to expand storage and some swap file for ram since it is arm based.I successfully installed in gl.inet flint2 where it runs flawless

**Comment by Ylianst** [comment]
I am not going to try on my wrt1900acs since it's my main router and don't want to mess with it. However, if I had a router with a bunch of headroom I would do it. My Home Assistant computer would really be where I would want MeshCentral to be, but that is a different story.
